pkg/accounting/rollup/rollup.go

```diff
@@ -43,7 +44,10 @@ func (r *rollup) Run(ctx context.Context) (err error) {
 		select {
 		case <-r.ticker.C: // wait for the next interval to happen
 		case <-ctx.Done(): // or the rollup is canceled via context
-			_ = r.db.Close()
+			err = r.db.Close()
```

Comment:
if rollup didn't start accountingdb, should it be closing it?

Reply:
isn't the close method just for disconnecting?